File: 프로그래머스1단계/두정수사이의합.cpp
#include <string>
#include <vector>

using namespace std;

// void swap(int a, int b){
//     int temp;
//     temp = a;
//     a = b;
//     b = temp;
// }

long long solution(int a, int b) {
    long long answer = 0;
    
    if(a > b){
        int temp;
        temp = a;
        a = b;
        b = temp;
    }
    
    for(int i = a; i <=b ; i++){
        answer += i;
    }
    
    return answer;
}

File: 프로그래머스1단계/두정수사이의합.java
import java.util.*;

class 두정수사이의합 {
    
    public long solution(int a, int b) {
        long answer = 0;
        int tmp = 0;
        if(a > b){
            tmp = a;
            a = b;
            b = tmp;
        }
        
        for(int i=a; i <= b; i++){
            answer += i;
        }
        
        return answer;
    }
    
}